src/model/services/mapi/applicationv1alpha1/key.ts

```diff
@@ -44,6 +45,12 @@ export function getAppCatalogEntryReadmeURL(
   return appCatalogEntry.metadata.annotations?.[annotationReadme];
 }
 
+export function getAppCatalogEntryLogoURL(
+  appCatalogEntry: IAppCatalogEntry
+): string | undefined {
+  return appCatalogEntry.metadata.annotations?.[annotationLogo];
```

Comment:
Do you think it's possible to move icon fallback inside this function?
like this:
```
   return appCatalogEntry.metadata.annotations?.[annotationLogo] ?? appCatalogEntry.spec.chart.icon;
```

Reply:
Good idea 👍 I'll make the changes